spec/feature/authenticated_user_spec.rb

```diff
@@ -194,4 +189,11 @@ def click_add_to_cart_link(category)
       end
     end
   end
+
+  def valid_user_logs_in
+    click_link_or_button "Log In"
+    fill_in 'session_email', :with => "rich.shea@gmail.com"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.